elections/README.md

```diff
@@ -0,0 +1,80 @@
+* [Kata Containers Elections](#kata-containers-elections)
+  * [Architecture Committee Elections](#architecture-committee-elections)
+      * [Election Dates](#election-dates)
+      * [Electorate](#electorate)
+      * [Candidacy](#candidacy)
+      * [Architecture Committee Responsibilities](#architecture-committee-responsibilities)
+      * [System](#system)
+
+
+
+# Kata Containers Elections
+
+This repository contains Kata Containers Elections reference documents
+and tooling to run elections.
+
+
+# Architecture Committee Elections
+
+
+## Election Dates
+
+The Architecture Committee elections take place every 6 months, with 3 seats
+up for election in September and 2 seats up for election in February.
+
+The election process begins 3 weeks prior to the election.
+- One week for candidates to submit their candidacy
```

Comment:
My inclination is to not put this in the documentation here, but that the corresponding election folder and all announcements about the election from the officials needs to be very clear about the time and date, to avoid the disappointment you mention. Thoughts?